src/main/java/seedu/address/logic/commands/DeleteGroupCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GROUP_NAME;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.group.Group;
+
+public class DeleteGroupCommand extends Command {
+    public static final String COMMAND_WORD = "delete group";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": deletes a group from the address book. "
+            + "Parameters: "
+            + PREFIX_GROUP_NAME + "GROUP_NAME ";
+    public static final String MESSAGE_SUCCESS = "Gerson deleted: %1$s";
+    public static final String MESSAGE_GROUP_DOES_NOT_EXIST = "This group does not exist in the address book";
+
+    private Group group;
+
+    /**
+     * Delete group command, delete based on name
+     * @param group group object to be deleted
```

Comment:
Should the comments and @param be seperated with a line in between?

Reply:
@marcuspeh yes I will make the corresponding changes.